docs/developer-docs/latest/developer-resources/content-api/content-api.md

```diff
@@ -1095,3 +1101,7 @@ OR
 If you only want to retrieve your draft entries, you can combine the `preview` mode and the `published_at` field.
 `GET /articles?_publicationState=preview&published_at_null=true`
 :::
+
+### Locale
+
+If the [Internationalization (i18n) plugin](/developer-docs/latest/development/plugins/i18n.md) is installed, the `locale` API parameter can be used to [get entries from a specific locale](/developer-docs/latest/development/plugins/i18n.md#getting-localized-entries-with-the-locale-parameter).
```

Comment:
isn't it `_locale` instead of `locale`?

Reply:
In my mind `locale` is the _name_ of the parameter and `_locale` is the _syntax_ that needs to be used in queries. Just like for `sort`, `start`, `limit`… and other API parameters.